frontend/src/lib/format/error.ts

```diff
@@ -1,11 +1,16 @@
 /**
  * Takes an errors object and returns a `string[]` with correct messages.
- * @param errors
+ * Filters out irrelevant error codes like "session_expired".
+ * @param errors The error response body.
+ * @returns A list of error messages.
  */
 export function collectErrors(errors: string | object): string[] {
   if (typeof errors === "string") {
     return [errors];
   }
-  const flatten = Object.values(errors || {}).flat();
+
+  const flatten = Object.values(errors || {})
+    .flat()
+    .filter((error) => error !== "session_expired");
```

Comment:
This only works for the "session_expired" code, I think we should just filter out any record with the key "code", also please add some documentation on why this is and update the test file with a testcase for it :)